### DISCUSSION TITLE
大型应用如何使用emp拆分？

### DISCUSSION THREAD

**Discussion by node-rookie** [empjs/emp#122] [Q&A]
emp如何做到像乾坤一样拆分大型应用为多个子应用，然后通过基座应用聚合所有子应用，提供类似同一个应用的使用体验？

**Comment by ckken** [comment]
路由为引导 组件模块为 页面单元即可

**Comment by ckken** [comment]
https://github.com/efoxTeam/emp/tree/main/projects/antd-project1

**Reply by troyzhxu** [reply]
“路由为引导 组件模块为 页面单元”，大神有没有文章 举例详细 讲讲  这句话 在 vue 项目下 怎么实施啊？
